src/cinder/Log.cpp

```diff
@@ -502,6 +521,64 @@ void LoggerSysLog::write( const Metadata &meta, const string &text )
 	syslog( sysLevel , "%s %s", meta.toString().c_str(), text.c_str() );
 }
 
+#elif defined( CINDER_MSW )
+
+// ----------------------------------------------------------------------------------------------------
+// MARK: - LoggerEventLog
+// ----------------------------------------------------------------------------------------------------
+
+LoggerEventLog::LoggerEventLog()
+{
+	char filename[MAX_PATH];
+	wchar_t wFilename[MAX_PATH];
+	string stem;
+
+	DWORD size = GetModuleFileNameA( NULL, filename, MAX_PATH );
+	if( size ) { 
+		boost::filesystem::path exePath(filename);
+		stem = exePath.stem().string();
+	} else {
+		app::Platform::get()->console() << "Could not determine application name, defaulting to 'CinderApp'" << endl;
+		stem = "CinderApp";
+	}
+
+	mbstowcs( wFilename, stem.c_str(), stem.size() + 1 );
+	mHLog = RegisterEventSourceW( 0, wFilename );
+
+	if( ! mHLog ) {
+		app::Platform::get()->console() << "RegisterEventSourceW() failed with " << GetLastError() << endl;
+	}
+}
+
+LoggerEventLog::~LoggerEventLog()
+{
+	if( mHLog ) {
+		CloseEventLog(mHLog);
```

Comment:
spacing (and same `::` prefix treatment), should look like `::CloseEventLog( mHLog );`